Error while checking build/O2/o2-dataflow for b82f0f7370e8a65a45600a92da2a25f447cccab5 at 2022-09-20 19:46:
```
## sw/BUILD/O2-latest/log
100% tests passed, 0 tests failed out of 429
 93/106 Test #107: test_Framework_test_SingleDataSource ....................***Failed    2.81 sec
[ERROR] pid 25302 (internal-dpl-injected-dummy-sink) crashed with 1
[ERROR] SEVERE: Device internal-dpl-injected-dummy-sink (25302) returned with 1
99% tests passed, 1 tests failed out of 102
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/AliceO2/9863/b82f0f7370e8a65a45600a92da2a25f447cccab5/build_O2_o2-dataflow/pretty.html).

--- a/Detectors/TRD/calibration/macros/plotVdriftExB.C
+++ b/Detectors/TRD/calibration/macros/plotVdriftExB.C
@@ -36,7 +36,7 @@ static std::array<std::vector<std::tuple<double, double, timePoint>>, 540> vmap;
 static std::array<bool, 540> good;                                               // Marks wether or not a chamber was 'good' for the entire duration.
 
 // Download the values and populate the map.
-void ccdbDownload(unsigned int runNumber, std::string ccdb, timePoint timeOffset)
+void ccdbDownload(unsigned int runNumber, std::string ccdb, timePoint queryInterval)
 {
   auto& ccdbMgr = o2::ccdb::BasicCCDBManager::instance();
   ccdbMgr.setURL("http://alice-ccdb.cern.ch/");
@@ -49,7 +49,7 @@ void ccdbDownload(unsigned int runNumber, std::string ccdb, timePoint timeOffset
   const auto endTime = grp->getTimeEnd();
   ccdbMgr.setURL(ccdb);
 
-  for (timePoint time = startTime; time < endTime; time += timeOffset) {
+  for (timePoint time = startTime; time < endTime; time += queryInterval) {
     ccdbMgr.setTimestamp(time);
     std::cout << "Getting vDriftExB for " << time << std::endl;
     auto calVdriftExB =
@@ -119,9 +119,9 @@ std::unique_ptr<TMultiGraph> draw(int i)
 // The times must be given in unix epoch ms format.
 // The default Run is 523677 for the test ccdb.
 // The offset for the next calibration is 15 Minutes.
-void plotVdriftExB(unsigned int runNumber = 523677, std::string ccdb = "http://ccdb-test.cern.ch:8080", timePoint timeOffset = 899991)
+void plotVdriftExB(unsigned int runNumber = 523677, std::string ccdb = "http://ccdb-test.cern.ch:8080", timePoint queryInterval = 900000)
 {
-  ccdbDownload(runNumber, ccdb, timeOffset);
+  ccdbDownload(runNumber, ccdb, queryInterval);
   find_good();
   print_good();
 

